draft-ietf-quic-transport.md

```diff
@@ -1461,127 +1461,126 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
-connection as it migrates, a client can use different connection IDs when
-sending from different addresses. The server can send NEW_CONNECTION_ID
-frames to provide the client with additional connection IDs to be used during
-migration ({{migration-linkability}}).
-
-Any time a new remote endpoint is detected, such as when a server
-receives a packet for a known connection ID from a different address, the remote
-endpoint MUST be validated ({{migration-validate}}).
-
-Any time a new local endpoint is used, such as when a client chooses
-to migrate a connection onto a different network interface, the path can be
-probed to check reachability ({{migration-probe}}). An endpoint may choose to
-probe a network path before migrating the connection, or it may choose to
-migrate the connection immediately. For example, a client that detects that its
-current network path is degraded may choose to probe alternate paths and only
-migrate if an alternate path is preferred over the current one. A client may
-also find that a path suddenly becomes inoperable, such as when a user
-unplugs a cable or other network state changes, and may elect to migrate
-without probing since it has no alternative if it wishes to maintain the
+QUIC connections are identified by 64-bit Connection IDs. QUIC's connection IDs
+allow connections to survive changes to the client's IP address and/or port -
+collectively referred to as the client's address - such as those caused by a
+client migrating to a new network or a NAT rebinding changing the publicly
+observable client address.
+
+A client migrating to a new network MUST use a new connection ID for packets
+sent from the new address if an additional connection ID has been provided by
+the server, see {{migration-linkability}}.  Using a new connection ID on
+migration makes it harder for passive observers to track connections migrating
+across networks. A client may be unaware of connection migration due to NAT
+rebinding and may consequently not change its connection ID on NAT rebindings.
+
+A server that receives a packet for a known connection ID from a new client
+address SHOULD validate this new address, see {{migration-validate}},
+
+
+### Path Validation and PMTU Verification
+
+PATH_CHALLENGE ({{frame-path-challenge}}) and PATH_RESPONSE
+({{frame-path-response}}) frames MAY be used for confirming that an endpoint
+owns an address, or to verify the PMTU to/from an address.
+
+An endpoint validates a peer's address by sending a PATH_CHALLENGE frame
+containing a payload that is hard to guess.  This frame MUST be sent in a packet
+to the new address.  Once a PATH_RESPONSE frame containing the same payload is
+received from the peer, the address is considered to be valid.  A PATH_CHALLENGE
+frame MUST contain at least 12 randomly generated octets {{?RFC4086}}, making
+the payload sufficiently hard to guess.
+
+An endpoint SHOULD not generate two PATH_CHALLENGE frames with the same payload.
```

Comment:
Based on below text, it seems like it is the latter case.

Reply:
I figured unique for all paths, since having the same random bytes used for two different paths can also get confusing, and because it's fairly trivial to not have them be the same. Do you think it needs clarifying? (I'll fix the SHOULD NOT)

Reply:
Nope, it's fine as is I think.

Reply:
I've changed some text here.

Reply:
Thanks, this looks good!